core/frontend/src/Viewport.ts

```diff
@@ -48,8 +48,7 @@ import { createRenderPlanFromViewport } from "./render/RenderPlan";
 import { RenderTarget } from "./render/RenderTarget";
 import { StandardView, StandardViewId } from "./StandardView";
 import { SubCategoriesCache } from "./SubCategoriesCache";
-import { DisclosedTileTreeSet, MapLayerImageryProvider, MapTileTreeReference, TileBoundingBoxes, TiledGraphicsProvider, TileTreeReference } from "./tile/internal";
-import { MapTiledGraphicsProvider } from "./tile/map/MapTiledGraphicsProvider";

```

Comment:
@pmconne is there a reason this file wasn't in the barrel? I added it to the barrel, but I'm not sure if I added it in the right place considering it could introduce a cyclic dependency

Reply:
Nope, just an oversight on somebody's part. Location in internal.ts looks ok.